test/spec/webfinger.js

```diff
@@ -0,0 +1,31 @@
+define(function(require) {
+  var util = require('../util/util');
+
+  describe('WEBFINGER', function () {
```

Comment:
lowercase is ok here. The others are capitalized because they're states in the `/authn` api